modules/shortcodes/bandcamp.php

```diff
@@ -220,7 +220,18 @@ function shortcode_handler_bandcamp( $atts ) {
 		$extraAttrs = '';
 	}
 
-	return "<iframe width='" . esc_attr( $width ) . "' height='" . esc_attr( $height ) . "' style='position: relative; display: block; width: " . esc_attr( $csswidth ) . '; height: ' . esc_attr( $cssheight ) . ";' src='" . esc_url( $url ) . "' allowtransparency='true' frameborder='0'" . $extraAttrs . '></iframe>';
+	$result  = '<iframe width="';
+	$result .= esc_attr( $width );
+	$result .= ' height=';
```

Comment:
I think we are missing the closing `"` here, and for the height as well. Maybe change it all into a `sprintf`?